feature/google/src/test/java/com/revenuecat/purchases/google/BillingWrapperTest.kt

```diff
@@ -379,9 +379,11 @@ class BillingWrapperTest {
 
         val upgradeInfo = mockReplaceSkuInfo()
         val productDetails = mockProductDetails(productId = productId, type = subsGoogleProductType)
+        val defaultOfferDetails = productDetails.subscriptionOfferDetails!![0]
         val storeProduct = productDetails.toStoreProduct(
-            productDetails.subscriptionOfferDetails!![0].subscriptionBillingPeriod,
-            productDetails.subscriptionOfferDetails!!
+            defaultOfferDetails.subscriptionBillingPeriod,
```

Comment:
i think we always want the billingPeriod to be that of the base plan... which makes me wonder if we should even pass it in, as that could be more error-prone? we could just use the offerdetails passed in to get the baseplan's period.

if we migrate to a factory, perhaps we can clean all this up though